rules/windows/command_and_control_remote_file_copy_powershell.toml

```
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2020/11/30"
+maturity = "production"
+updated_date = "2020/11/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies powershell.exe being used to download an executable file from a remote destination.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Remote File Download via Powershell"
+risk_score = 43
+rule_id = "33f306e8-417c-411b-965c-c2812d6d3f4d"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control"]
+type = "eql"
+
+query = '''
+
+sequence by host.id, process.entity_id with maxspan = 30s
+  [network where process.name : "powershell.exe" and network.protocol == "dns" and
+   not dns.question.name : ("localhost", "*.microsoft.com", "*.azureedge.net", "*.powershellgallery.com", "*.windowsupdate.com", "metadata.google.internal") and 
+   not user.domain : "NT AUTHORITY"]
+  [file where event.type == "creation" and file.extension : ("exe", "dll", "ps1", "bat") and 
+   not file.path : "C:\\Windows\\Temp\\__PSScriptPolicy*.ps1"]
```

```suggestion
sequence by host.id, process.entity_id with maxspan=30s
  [network where process.name : "powershell.exe" and network.protocol == "dns" and
   not dns.question.name : ("localhost", "*.microsoft.com", "*.azureedge.net", "*.powershellgallery.com", "*.windowsupdate.com", "metadata.google.internal") and 
   not user.domain : "NT AUTHORITY"]
  [file where event.type == "creation" and file.extension : ("exe", "dll", "ps1", "bat") and 
   not file.path : "C:\\Windows\\Temp\\__PSScriptPolicy*.ps1"]
```

Those leading wildcards for `dns.question.name` are not great, but I don't think there is any better way atm

---

yeah, its noise vs performance for powershell download activity